recipe/bld.bat

```diff
@@ -1,2 +1,40 @@
+:: Doesn't include gmock or gtest. So, need to get these ourselves for `make check`.
+git clone -b release-1.7.0 git://github.com/google/googlemock.git gmock
```

Comment:
Unfortunately, [no]( https://github.com/google/googlemock/blob/release-1.7.0/Makefile.am#L206-L208 ) for [either]( https://github.com/google/googletest/blob/release-1.7.0/Makefile.am#L296-L298 ). Future releases of these may actually be better about installing. Some Google libraries bundle them in releases. However, we weren't able to find a good release with Python and C++ that had these bundled. So, we had to do it ourselves. This is their [recommended]( https://github.com/google/protobuf/tree/v3.0.0-beta-2/cmake#getting-sources ) method.